nvflare/app_common/pt/pt_fed_utils.py

```diff
@@ -18,5 +18,7 @@
     f"This module: {__file__} is deprecated. Please use nvflare.app_opt.pt.", category=FutureWarning, stacklevel=2
```

Comment:
Should the warning say `nvflare.app_opt.pt.model_persistence_format_manager`?

Reply:
previously this `model_persistence_format_manager` is put inside `pt_fed_utils.py`
but right now in app_opt I separate them, what do you think should be the correct warning?

Reply:
I see. Please ignore my comment. The current warning should be fine then.